profiles/kentik_snmp/hpe/hpe-synergy.yml

```diff
@@ -0,0 +1,54 @@
+#https://mibs.observium.org/mib/SYNERGY100G-HPE-CPU-UTIL-MIB/
+#https://mibs.observium.org/mib/SYNERGY100G-HPE-NLP-MIB/#syn100GhpeNLPModuleConfig
+---
+extends:
+  - system-mib.yml
+  - if-mib.yml
+  - ip-mib.yml
+
+sysobjectid:
+  - 1.3.6.1.2.1.11.5.7.5.9    # VC SE 100Gb F32 Module
+
+
+provider: kentik-server-chassis
+
+metrics:
+  - MIB: HPE-SYNERGY-MIB
+    symbols:
+      - name: syn100GhpeSwitchMaxCPUThreshold
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.1.1
+      - name: syn100GhpeSwitchMinCPUThreshold
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.1.2
+      - name: syn100GhpeSwitchAverageCPUUtilization  
+        OID: 1.3.6.1.4.1.11.5.7.5.9.1.4130.2.1
+        tag: CPU
```

Comment:
you should leave the 60s interval on here to ensure the metric is available for anomaly detection